app/code/community/Hackathon/EmailPreview/Model/Mail/Type/AdminPasswordEmail.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+class Hackathon_EmailPreview_Model_Mail_Type_AdminPasswordEmail
+{
+    const TYPE_FORGOT_ADMIN_PASSWORD = 'test_forgot_admin_password_email_template';
+    
+    /**
+     * @param Varien_Event_Observer $observer
+     * @return Hackathon_EmailPreview_Model_Mail_Type_OrderEmail
+     */
+    public function hackathonEmailpreviewRenderEmailBefore(Varien_Event_Observer $observer)
+    {
+        if (!in_array($observer->getEvent()->getData('templateType'), array(self::TYPE_FORGOT_ADMIN_PASSWORD))) {
+            return $this;
+        }
+
+        //@todo change logic here
```

Comment:
What would be necessary to get this todo done (question also applies for the other @todo comments)?

Reply:
same here. quite a long time, since i changed the code  :)